### DISCUSSION TITLE
「委托」空洞取景

### DISCUSSION THREAD

**Discussion by chesanwen** [share121/inter-knot#5327] [绝区零]
![6071d5e75a58c2c16268b88bff38d15f](https://github.com/user-attachments/assets/0dff2283-0d6f-45b6-8374-1491144cdce6)
《向更深处进发》电影于8.16开机，部分片段需在c28空洞拍摄，急需向导。
有意者请联系我们，报酬面谈，非诚勿扰，感谢配合。

**Comment by aomengxin** [comment]
我非诚，我就扰

**Comment by TianyiLuo0721** [comment]
这个委托我接了！

**Comment by lifelinemrx** [comment]
这个委托我接了！

**Comment by 189469** [comment]
这个委托我接了!

**Comment by bhaaaag** [comment]
这个委托我接了

**Comment by Ruobai118** [comment]
这个委托我接了！

**Comment by Kazuto-Kiritani** [comment]
传奇绳匠接了

**Comment by fengganchun** [comment]
这个委托我接了！

**Comment by Robin-txdy** [comment]
这个委托我接了<img src="https://bbs-static.miyoushe.com/static/2024/10/24/b364c5784e16b15f53af9db191666b4a_936313514384740306.png" alt="TADA" width="75" >

**Comment by PB4437** [comment]
啊～已经聚集了那么多绳匠了～全部抓起来！

**Comment by phaethon372** [comment]
这委托我接了
